internal/cf/client_test.go

```diff
@@ -337,6 +337,38 @@ var _ = Describe("CF Client tests", Ordered, func() {
 			Expect(server.ReceivedRequests()[3].URL.Path).To(Equal(instancesURI))
 		})
 
+		It("should re-create the refresh token after configured RefreshTokenAutoRenewalInterval", func() {
+			cfgZero := config.Config{
+				RefreshTokenAutoRenewalInterval: time.Duration(time.Second * 0),
+			}
+			spaceClient, err := NewSpaceClient(OrgName, url, Username, Password, &cfgZero)
+			Expect(err).To(BeNil())
+
+			spaceClient.GetInstance(ctx, map[string]string{"owner": Owner})
+			spaceClient, err = NewSpaceClient(OrgName, url, Username, Password, &cfgZero)
+			Expect(err).To(BeNil())
+			spaceClient.GetInstance(ctx, map[string]string{"owner": Owner})
+
+			Expect(server.ReceivedRequests()).To(HaveLen(6))
```

Comment:
You are expecting 6 requests, but checking indexes 0,1,2,4,5.
Do you also want to check index 3 ?